theme/themes/pxt/globals/site.variables

```diff
@@ -358,6 +358,8 @@
 @errorListInnerHeight: (@errorListHeight - @errorListHeaderHeight);
 @errorListMessageHoverColor: rgba(0, 0, 0, 0.1);
 @errorListBorderColor: #e5e5e5;
-@errorListOdd: @homeScreenBackground;
```

Comment:
This was assuming homeScreenBackground === monaco editor background, but that's only true if the editor specifically sets them the same; monaco colors are set like this https://github.com/microsoft/pxt-microbit/blob/master/pxtarget.json#L509 and they've been set to match for arcade & microbit but not minecraft (which has a light homescreen / dark editor).

As written this changes to just lighten / darken but it goes towards gray more than the current behavior with lesses darken; current arcade:

![image](https://user-images.githubusercontent.com/5615930/89687090-62b58e80-d8b4-11ea-89d9-a71693b16fef.png)

this change: 

![image](https://user-images.githubusercontent.com/5615930/89687061-55989f80-d8b4-11ea-8a0c-e80ad68ad286.png)

I personally prefer the grayer one as it matches with the highlight color, but can add a variable to maintain hue like it currently does if preferred ~